crates/matrix-sdk-crypto/src/machine/mod.rs

```diff
@@ -2457,6 +2499,29 @@ fn sender_data_to_verification_state(
     }
 }
 
+/// Real implementation for `OlmMachine::mark_all_tracked_users_as_dirty`.
```

Comment:
Let's at least put this method under the `IdentityManager`.

Reply:
Done.